packages/rest/src/client/RestClient.ts

```diff
@@ -16,6 +16,10 @@ import { AbstractRestClient } from "./AbstractRestClient";
 import { JSONUtils } from "../../../utilities";
 import { Readable, Writable } from "stream";
 import { ITaskWithStatus } from "../../../operations";
+import { IFullResponseOptions } from "./doc/IFullResponseOptions";
+import { IRestClientResponse } from "./doc/IRestClientResponse";
+import { IOptionsFullRequest } from "./doc/IOptionsFullRequest";
+import { CLIENT_PROPERTY } from "./types/AbstractRestClientProperties";
 
```

Comment:
Would you be up for updating the remaining functions so that they don't call the recently deprecated `performRest` function?

Reply:
Sure I can update them.